guides/user/kubernetes-v2/deploy-manifest/index.md

```diff
@@ -0,0 +1,173 @@
+it directly in the pipeline the manifest specification.
+
+{%
+  include
+  figure
+  image_path="./in-pipeline.png"
+  caption="Notice that by selecting \"Text\" as the __Manifest Source__, we get
+  to enter the Manifest YAML by hand."
+%}
+
+Of course, if you are generating your pipeline definitions rather than entering
+them into the UI, the stage definition would look more like:
+
+```json
+{
+  "name": "Deploy my manifest",   // human-readable name
+  "type": "deployManifest",       // tells orchestration engine what to run
+  "account": "nudge",             // account (k8s cluster) to deploy to
+  "cloudProvider": "kubernetes",
+  "source": "text",
+  "manifest": {
+                                  // manifest contents go here
+  },
+  "moniker": {                    // used to specify app & cluster for grouping
+                                  // resources in UI
+    "app": "xnat",
+    "cluster": "c7",
+  }
+}
+```
+
+### Dynamically specified manifests
+
+If you are storing your manifests outside of Spinnaker's pipeline repository,
+or want a single deploy stage to be able to deploy a variety of manifests, you
+can specify your manifest using an [Artifact](/reference/artifacts).
+
+The idea is: artifacts in Spinnaker allow you to reference remote, deployable
+resources. In our case, artifacts will have to reference a text file
+(containing a Manifest specification) stored in GitHub, or an object store
+(like GCS).
+
+There are already some guides describing how to trigger off of changes to these
+artifacts, and supply them to pipelines. See:
+
+* [Consuming GitHub Artifacts](/guides/user/artifacts/github)
+* [Consuming GCS Artifacts](/guides/user/artifacts/gcs)
+
+Assuming you have declared an expected artifact upstream to your Deploy
+manifest stage, you can reference it in the Deploy configuration:
+
+{%
+  include
+  figure
+  image_path="./in-artifact.png"
+  caption="Notice that by selecting \"Artifact\" as the __Manifest Source__, we
+  get to pick which upstream artifact to deploy."
+%}
+
+> __☞ Note__: Based on how you configured your artifact accounts, make sure that
+> the __Artifact Account__ field matches an artifact with permission to
+> download your manifest.
+
+Keep in mind that the artifact bound in the upstream stage can match multiple
+incoming artifacts. If instead we had configured it to listen to changes using
+a regex matching `.*\.yml`, it would bind any YAML file that changes in your
+artifact source, and deploy it when it reaches your Deploy stage.
+
+## Overriding Artifacts
+
+In general, when we deploy changes to our infrastructure, the majority of
+changes will come in the form of a new Docker image, or perhaps a feature-flag
+change in a ConfigMap. For this reason, we have first-class mechanisms for
+easily overriding the version a:
+
+* Docker Image
+* Kubernetes ConfigMap
+* Kubernetes Secret
+
+When any of these objects exists in the pipeline context from an upstream
```

Comment:
s/exists/exist

Reply:
Wait, in the next sentence you say "...automatically try to inject it..."
Is it necessarily one object, or can more than one exist? If only one, then I'd say change the previous sentence to "When one of these objects exists..."